roles/filetree_create/README.md

```diff
@@ -27,6 +27,10 @@ The following variables are required for that role to work properly:
 | `organization`| N/A | no | str | Default organization for all objects that have not been set in the source controller.|
 | `export_related_objects` | False | no | bool | Whether to export related objects (job templates related to certain workflows and the projects associated with these job templates) when a single JT or a single WFJT are being exported. |
 | `update_project_state` | False | no | bool | Whether the project should be updated after import to the target controller. |
+| `object_id` | N/A | no | int | Specifying object id with object type exports the access list of object |
```

Comment:
this is not a variable defined by user. It is set from task file execution. I think we should remove it from README.